app/tests/sample_data_test/collections/valid_collection_set_1.json

```diff
@@ -0,0 +1,30 @@
+{
+  "crs": [
+    "http://www.opengis.net/def/crs/OGC/1.3/CRS84"
+  ],
```

Comment:
crs should be a read only field and not valid in POST/PUT request

Reply:
@ltshb 
`  "crazy-stuff": [
    "adding this as well"
  ],`

this works as well. Shall we explicitly give a error, when read only fields have been added to the request?

Reply:
That's a good point. @boecklic what is your opinion ? Should we still silently ignore unknown attributes or return a 400 BAD REQUEST ?

Advantage of returning 400:

- Clear usage of API, we will not have people complaining that the API is buggy when trying to set the CRS and nothing is happening

Advantage of silently ignoring unknown/invalid attribute:

- In the use case where we would switch between new/old version of the API with new attributes, we will get less troubles. Not sure if this use case will ever happen.

Reply:
In any case I would remove the crs from the data sample and if we need to return a 400 in this case, this should be done in a separate PR.

Reply:
@ltshb In any case I will remove the crs from the data sample